### PR TITLE
New alumni pages

### PR DESCRIPTION
**Pull request by cedricmenteau**
@ssaunier @Papillard Voici les pages `/alumni`, `/projects` & `/shop`. Toutes responsives et testées sur Chrome / Safari / Firefox. Le CSS et quelques components ont besoin d'une refacto mais j'incluerai ça dans la suite du chantier `www` comme vu avec Boris.

www_new_pages_opti

### CODE REVIEW

File: app/controllers/stories_controller.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class StoriesController < ApplicationController
   def show
-    @story = @client.story(params[:id])
+    @story = story.story(params[:id])
```
Comment (ssaunier) [inline app/controllers/stories_controller.rb]: ?

File: app/views/pages/shop.html.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+<h1>Le Wagon's shop<small>Because coders got style</small></h1>
+
+<div class="container">
+  <div id="myShop"><a href="//shop.spreadshirt.fr/lewagon">lewagon</a></div>
+</div>
+
+<script>
+  var spread_shop_config = {
+    shopName: 'lewagon',
+    locale: 'fr_FR',
```
Comment (ssaunier) [inline app/views/pages/shop.html.erb]: Can you try to put 

```js
locale: '<%= I18n.locale %>'
```

File: app/views/pages/shop.html.erb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    prefix: '//shop.spreadshirt.fr',
+    baseId: 'myShop'
+  };
+</script>
+
+<script type="text/javascript" src="//shop.spreadshirt.fr/shopfiles/shopclient/shopclient.nocache.js"></script>
+
+<% content_for :after_js do %>
+  <script>
+    setTimeout(() => {
```
Comment (ssaunier) [inline app/views/pages/shop.html.erb]: What is that?

File: config/routes.rb [hunk 4]
```diff
@@ -72,9 +72,11 @@
     get "employers", to: "pages#employers", template: "employers", as: :employers
     get "tv", to: "pages#tv", template: "tv", as: :tv
     get "alumni" => "students#index", as: :alumni
+    get "projects" => "projects#index", as: :projects
     get "blog", to: 'posts#index', as: :blog
     get "blog/:slug", to: 'posts#show', as: :post
     get "live", to: 'pages#live', as: :live
+    get "shop" => "pages#shop", as: :shop
```
Comment (ssaunier) [inline config/routes.rb]: Hmm, we should localise this shop, so nest this route under the locale block.

File: app/views/shared/_footer.html.erb [hunk 5]
```diff
@@ -92,6 +92,9 @@
             <a href="/blog">Blog</a>
           </div>
           <div>
+            <a href="/shop">Shop</a>
```
Comment (ssaunier) [inline app/views/shared/_footer.html.erb]: That should be a `link_to` using a path with the locale (like the press link below). It's using meta-programming with the `send` method.

File: config/routes.rb [hunk 6]
```diff
@@ -72,9 +72,11 @@
     get "employers", to: "pages#employers", template: "employers", as: :employers
     get "tv", to: "pages#tv", template: "tv", as: :tv
     get "alumni" => "students#index", as: :alumni
+    get "projects" => "projects#index", as: :projects
```
Comment (ssaunier) [inline config/routes.rb]: Same remark here, should be localized.

**Comment by ssaunier** [comment]
Amazing work 👍 🎉 👏

File: app/controllers/stories_controller.rb (hunk 1 above)
Comment (ssaunier) [inline app/controllers/stories_controller.rb]: Is this controller still used?

File: app/views/pages/shop.html.erb (hunk 3 above)
Comment (cedricmenteau) [inline app/views/pages/shop.html.erb]: I need to wait for the Spreadshirt JS to load then remove and append a new link to keep navigate into our website.

File: app/controllers/stories_controller.rb (hunk 1 above)
Comment (cedricmenteau) [inline app/controllers/stories_controller.rb]: I'm not using it as my stories go through `students_controller.rb`. I rolled it back to its previous setup. Waiting for spring cleaning!

File: app/views/pages/shop.html.erb (hunk 2 above)
Comment (cedricmenteau) [inline app/views/pages/shop.html.erb]: Not working. Still in english (even with DISABLE_CACHE).

**Comment by cedricmenteau** [comment]
@ssaunier Modifications ✅

**Pull request merged**